### PR TITLE
Skip topology update on set bond value

### PR DESCRIPTION
**Pull request by Pangoraw**
The topology will be unchanged because the update will only affect the bind
value. Therefore, we can skip the topology update to speed up the process. :zap:

### CODE REVIEW

**Comment by Pangoraw** [comment]
hmm looks like something is failing on the frontend test, i will take a look :eyes:

File: src/evaluation/Run.jl [hunk 1]
```diff
@@ -146,7 +146,7 @@ will_run_code(notebook::Notebook) = notebook.process_status != ProcessStatus.no_
 ###
 
 "Do all the things!"
-function update_save_run!(session::ServerSession, notebook::Notebook, cells::Array{Cell,1}; save::Bool=true, run_async::Bool=false, prerender_text::Bool=false, kwargs...)
+function update_save_run!(session::ServerSession, notebook::Notebook, cells::Array{Cell,1}; save::Bool=true, prerender_text::Bool=false, kwargs...)
```
Comment (pankgeorg) [inline src/evaluation/Run.jl:149]: Hey!! I may be missing something, but where is `run_async` now?

Comment (Pangoraw) [reply src/evaluation/Run.jl]: > Hey!! I may be missing something, but where is `run_async` now?

It was part of the `kwargs` array, which are then passed to `maybe_run_async!`. 
I [put it back](https://github.com/fonsp/Pluto.jl/compare/f9bad8ff21d61afe146a4e368a636df7bf85b152..682d085d86648c827622b510881449364d7cc2d8) to be more explicit :smiley:

File: src/evaluation/Run.jl [hunk 2]
```diff
@@ -181,21 +181,26 @@ function update_save_run!(session::ServerSession, notebook::Notebook, cells::Arr
...
-		end
+		maybe_run_async!(session, notebook, old, new, to_run_online; run_async=run_async, kwargs...)
 	end
 end
 
 update_save_run!(session::ServerSession, notebook::Notebook, cell::Cell; kwargs...) = update_save_run!(session, notebook, [cell]; kwargs...)
 update_run!(args...) = update_save_run!(args...; save=false)
 
 
+maybe_run_async!(session::ServerSession, notebook::Notebook, to_run::Vector{Cell}; kwargs...) =
```
Comment (fonsp) [inline src/evaluation/Run.jl:192]: Can we change this to `run_maybe_async!`?

**Review by fonsp — CHANGES_REQUESTED** [review]
-

File: src/evaluation/Run.jl (hunk 2 above)
Comment (Pangoraw) [reply src/evaluation/Run.jl]: :heavy_check_mark: Renamed and moved just after `run_reactive!`

**Review by fonsp — APPROVED** [review]

**Pull request merged**